assessment/assessment_engine.go

```diff
@@ -67,11 +67,10 @@ func initializeCollectors(conv *internal.Conv, sourceProfile profiles.SourceProf
 
 func performSchemaAssessment(collectors assessmentCollectors) (utils.SchemaAssessmentOutput, error) {
 	schemaOut := utils.SchemaAssessmentOutput{}
-	schemaOut.TableNames = collectors.infoSchemaCollector.ListTables()
-	schemaOut.ColumnNames = collectors.infoSchemaCollector.ListColumns()
-	schemaOut.IndexNameAndType = collectors.infoSchemaCollector.ListIndexesAndTypes()
+	schemaOut.SourceTableDefs, schemaOut.SpannerTableDefs = collectors.infoSchemaCollector.ListTables()
+	schemaOut.SourceIndexDef, schemaOut.SpannerIndexDef = collectors.infoSchemaCollector.ListIndexes()
 	schemaOut.Triggers = collectors.infoSchemaCollector.ListTriggers()
-	schemaOut.ColumnAssessmentOutput = collectors.infoSchemaCollector.ListColumnDetails()
+	schemaOut.SourceColDefs, schemaOut.SpannerColDefs = collectors.infoSchemaCollector.ListColumnDetails()
```

Comment:
nit: the collector returns details, and the assessment reads as definitions. Let's use common terminology for the same thing.